benchmark_runner/main/update_data_template_yaml_with_environment_variables.py

```diff
@@ -15,9 +15,13 @@ def update_environment_variable(dir_path: str, yaml_file: str, environment_varia
     """
     if not environment_variable_dict:
         environment_variable_dict = environment_variables.environment_variables_dict
-    oc = OC(kubeadmin_password=environment_variable_dict.get('kubeadmin_password', ''))
-    # set prom token
-    environment_variable_dict['prom_token'] = oc.get_prom_token()
+    # Allow faking the Prometheus token so that we can generate deterministic test files
+    if 'prom_token_override' in environment_variable_dict:
+        environment_variable_dict['prom_token'] = environment_variable_dict.get('prom_token_override', '')
```

Comment:
This only gets set manually if we're running the unit tests; otherwise the existing logic is used unchanged.  I don't believe anything needs to be added to `test_environment_variables.py` here; all of the integration tests should run unchanged.

Reply:
OK. got it